Automattic/woocommerce-payments
# Migrate the charges data layer to TypeScript

Fixes #1272 

Depends on #1263.

Adds types to most things in `client/data/charges`. Unfortunately we won't see everything come together neatly until the whole data store has been migrated, so some of the type information is still a bit limited.

#### Changes proposed in this Pull Request

* Migrate the charges data store to TypeScript.









#### Testing instructions

- `rm -rf dist`
- `npm ci`
- `npm run test:js`
- Make sure tests run successfully
- `npm run watch`
- Make sure payment details screens load properly.





-------------------
